server/utils/workspace/workspace_utils.py

```
@@ -168,12 +168,13 @@ def update_workspace_aws_role(
         raise
 
 
-def get_workspace_by_id(workspace_id: str) -> Optional[Dict[str, Any]]:
+def get_workspace_by_id(workspace_id: str, user_id: str = None) -> Optional[Dict[str, Any]]:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)